### DISCUSSION TITLE
movie_retain parameter

### DISCUSSION THREAD

**Discussion by Singman33** [Motion-Project/motionplus#91] [General]
Hi,

Very simple question : can you explain the parameter "movie_retain" when set to "secondary" ?
I see in code that Motionplus is trying to remove some files but that's all.

Thanks

**Comment by Mr-Dave** [comment]
Motionplus allows for use of a secondary detection method.  This uses OpenCV and various models that a user may add.  If the secondary motion detection is triggered, then retain the movie.  If secondary motion detection does not get triggered, delete the movie.
